### PR TITLE
add an option for grayscale - close mash/node-imagemagick-native#70

### PR DESCRIPTION
**Pull request by dcharbonnier**
Images are required for the readme.
grayscale_before
grayscale_after

Let me know if this pull request need any change !
Thx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,6 +185,7 @@ The `options` argument can have following values:
         format:         optional. output format, ex: 'JPEG'. see below for candidates
         filter:         optional. resize filter. ex: 'Lagrange', 'Lanczos'.  see below for candidates
         blur:           optional. ex: 0.8
+        grayscale:      optional. true or false
```
Comment (timtty) [inline README.md]: Note: Listed as bool: true or false

**Comment by timtty** [comment]
Tested and works. Seems to be based on an older release.

**Comment by dcharbonnier** [comment]
if this can be integrated too I will create a new PR @elad

**Comment by elad** [comment]
@dcharbonnier go for it

**Comment by dcharbonnier** [comment]
cf. #149

**Pull request closed without merging**